src/containers/SearchResult.jsx

```diff
@@ -46,19 +44,27 @@ const Dot = styled.div`
   border-radius: 50%;
   background-color: #999999;
 `
+
 const ChooseFlightSection = styled.div`
+  margin-top: -3px;
```

Comment:
I think we need to add some margin between route items, so this value should be positive, around `1em`